.github/workflows/auth-release.yml

```diff
@@ -163,17 +102,16 @@ jobs:
             - name: Build desktop app
               run: |
                   flutter config --enable-linux-desktop
-                  # dart pub global activate flutter_distributor
-                  dart pub global activate --source git https://github.com/prateekmedia/flutter_distributor --git-ref develop --git-path packages/flutter_distributor
-                  # Run below command if it is a beta or nightly
-                  if [[ ${{ github.ref }} =~ beta|nightly ]]; then
-                    flutter_distributor package --platform=linux --targets=pacman --skip-clean
-                    mv dist/**/*-*-linux.pacman artifacts/ente-${{ github.ref_name }}-x86_64.pacman
```

Comment:
We don't need this anymore?

Reply:
Yes, pacman is not stable yet. It doesn't work simply put as of its current implementation.